script/Arcueid/Day 4/04_00_ARC04_6_2B.txt

```diff
@@ -407,7 +407,7 @@ Wouldn't it be easier to just do it all by yourself without creating these serva
 [sha:e435916eca9d3de99b264c5b381753bc5ef102e4]{
 -- Page 501, Offset 6940.
 -- どれほどの超抜能力、どれほどの<存在規模|ライフスケール>をもった吸血鬼でも、<単身|ひとり>では生きられない。
-No matter how much supernatural power they have, or how large their scope of existence is, vampires cannot survive on their own.
+No matter how much superpower they have, or how large their Lifescale is, vampires cannot survive on their own.
```

Comment:
Seems a bit weird to have a _quantity_ of a superpower? I feel like they're usually referred to in terms of extent, potency?
> No matter how incredible their superpowers may be, or ...
> No matter how powerful they are, or ...

Reply:
I'm fine with the former